pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -40,24 +47,24 @@ def __init__(
         self.dicom_timeout = dicom_timeout
 
     @property
-    @abstractmethod
     def aet(self) -> str:
         """Application entity title (AET) of this Orthanc instance"""
+        return self._aet
```

Comment:
I thought this made it slightly easier to set the `aet` value for each instance of the class, but happy to change it back

Reply:
nah seems reasonable to me